server/src/main/java/org/elasticsearch/action/admin/cluster/allocation/DesiredBalanceResponse.java

```diff
@@ -35,19 +36,23 @@
 public class DesiredBalanceResponse extends ActionResponse implements ChunkedToXContentObject {
 
     private static final TransportVersion CLUSTER_BALANCE_STATS_VERSION = TransportVersion.V_8_7_0;
+    private static final TransportVersion CLUSTER_INFO_VERSION = TransportVersion.V_8_8_0;
```

Comment:
I wonder if it is worth also backporting to 8.7?

Reply:
It'd certainly be nice to have ASAP but it's not really a bugfix so I think we shouldn't backport.